haproxy/README.md

```diff
@@ -54,7 +54,8 @@ create your own configmap and values.
 | `sidecarContainers` | extra sidecar containers to add to the deployment | `[]`
 | `haproxy.frontendPort` | Port for the Proxy to listen on the frontend | 30636
 | `haproxy.config`       | Suffix of the used config-map | ldap-tls
-| `haproxy.check.existingSecret.name` | Use existing secret with authentication credentials for check commands.  | 
+| `haproxy.check.existingSecret.name` | Use existing secret with authentication credentials for check commands. |
+| `haproxy.securityContext` | Security context for the pod to be used | `{}`
```

Comment:
just cosmetic, but maybee swap the lines? so both existingSecret.name and existingSecret.key are in a row?